contents/blog/features-sell.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2022-10-20
+title: "Is this the death of benefits selling?"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+Features tell, benefits sell. Well, they used to.
+
+Marketing lore tells you that you must focus on the _why_, not the what. "No one buys a CRM, they buy a way to increase their revenue" - that sort of thing.
+
+I'm sure this worked great when it was invented. But now, it feels tired and out-of-touch to end users. I think developers are especially sensitive to vague language when evaluating products.
+
+It makes me feel like I'm being sold to, by someone who doesn't know what they're talking about.
+
+Worse, benefits-heavy and feature-light language makes it hard to know what you're actually buying. I can evaluate for myself what benefits something brings, I want to see how it works, and how it compares to other products, instead.
+
+Highlighting benefits makes sense, as a backup in case I don't get it. For executives who don't really use the software, and who certainly don't implement it - go nuts. For everyone else, benefits-oriented language is a poor user experience - it gets in the way of explaining the product.
```

Comment:
```suggestion
Highlighting benefits makes sense, as a backup in case I don't get it. For executives who don't use the software, and certainly don't implement it - go nuts. For everyone else, benefits-oriented language is a poor user experience - it gets in the way of explaining the product.
```